__test__/components/navbar.spec.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import NavBar, { AuthButtons } from '@components/NavBar';
+
+const mockState = menuOpen => {
+  const setState = jest.fn();
+  const useStateSpy = jest.spyOn(React, 'useState');
+  useStateSpy.mockImplementation(() => [true, setState]);
+};
+
+describe('<NavBar />', () => {
+  let wrapper;
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should test when user is not logged in and menu is clicked', () => {
+    mockState(false);
+    wrapper = mount(
+      <Router>
+        <NavBar isLoggedIn={false} />
+      </Router>
+    );
+    wrapper.find('#hamburger').simulate('click');
+    mockState(true);
+    expect(wrapper.find('#hamburger').hasClass('active')).toEqual(true);
+  });
+
+  it('should test when user is logged in', () => {
```

Comment:
```it displays something when the *isLoggedIn* prop is set to true```

Reply:
Your tests should be about what the UI displays based on application state.

So, in this case, it should test that the avatar is rendered when `isLoggedin = true`, but not rendered when set to `false`. It also renders the `AuthButtons` when set to `false` but not when set to `true`